<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>946</width>
    <height>498</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>946</width>
    <height>498</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Журнал контроля самоподготовки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>931</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Данные начальника курса</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>351</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер учебной группы</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество по списку</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество на лицо</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Примечание</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Данные для курсового офицера</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Выбрать интервал посещения:</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>С:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>по:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Определить</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>911</width>
        <height>281</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер учебной группы</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество по списку</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество на лицо</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Примечание</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Добавления записи</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>331</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавление записи</string>
      </property>
      <widget class="QDateEdit" name="dateEdit_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер учебной группы:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество по списку:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество на лицо:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите список:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>311</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>312</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>313</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>314</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Выберите причину отсутствия:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Наряд</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Санчасть</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Прогул</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите список:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>240</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Наряд</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Санчасть</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Прогул</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Выберите причину отсутствия:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите список:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>300</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>270</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Наряд</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Санчасть</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Прогул</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Выберите причину отсутствия:</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>581</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер учебной группы</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество по списку</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество на лицо</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Примечание</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>График пропусков</string>
     </attribute>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Наряды</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>261</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>261</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>330</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Санчасть</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>330</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Прогулы</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>461</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Подключение к серверу:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP-адрес:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>5555</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Вывести отчет:</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>С:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>по:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Вывести</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
